pom.xml

```diff
@@ -119,7 +119,7 @@
     <dependency>
       <groupId>uk.gov.ons.ctp.integration.common</groupId>
       <artifactId>event-publisher</artifactId>
-      <version>0.0.23</version>
+      <version>0.0.24-SNAPSHOT</version>
```

Comment:
I guess that you will need to change this after you have merged your PR for event-publisher. I will look at that one first.